src/AppBundle/Model/Event.php

```diff
@@ -221,6 +215,18 @@ public function getCacheKey()
         return (string)$this->id;
     }
 
+    /**
+     * Is the Event valid? If false, statistics will be able to be generated.
+     * @return bool
+     */
+    public function isValid()
+    {
+        return $this->wikis->count() > 0 &&
+            $this->start !== null &&
```

Comment:
Ther'es also 'event-in-future' which says that the statistics can't be done because the event is in the future -- do we need to add that test in the validity here?